src/Compilers/CSharp/Test/EndToEnd/EndToEndTests.cs

```diff
@@ -115,7 +115,7 @@ bool runTestAndCatch(int depth)
         // This test is a canary attempting to make sure that we don't regress the # of fluent calls that 
         // the compiler can handle. 
         [WorkItem(16669, "https://github.com/dotnet/roslyn/issues/16669")]
-        [ConditionalFact(typeof(WindowsOrLinuxOnly)), WorkItem(34880, "https://github.com/dotnet/roslyn/issues/34880")]
+        [ConditionalFact(typeof(WindowsOrLinuxOnly), AlwaysSkip = "https://github.com/dotnet/roslyn/issues/67912"), WorkItem(34880, "https://github.com/dotnet/roslyn/issues/34880")]
```

Comment:
Didn't mean for this skip to flow into 17.8. The was fixed in 17.7.

```suggestion
        [ConditionalFact(typeof(WindowsOrLinuxOnly)), WorkItem(34880, "https://github.com/dotnet/roslyn/issues/34880")]
```